.github/workflows/replay-verify.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
       BACKUP_CONFIG_TEMPLATE_PATH: terraform/helm/fullnode/files/backup/gcs.yaml
       # workflow config
-      RUNS_ON: "high-perf-docker-with-local-ssd"
+      RUNS_ON: "high-perf-docker-with-local-ssd-replay-verify"
```

Comment:
`high-perf-docker-with-local-ssd` is only used for replay-verify so you don't need a new runner pool.